php/WP_CLI/Runner.php

```diff
@@ -258,16 +259,20 @@ public function find_command_to_run( $args ) {
 				if ( count( $cmd_path ) > 1 ) {
 					$child = array_pop( $cmd_path );
 					$parent_name = implode( ' ', $cmd_path );
+					$suggestion = $this->get_subcommand_suggestion( $child, $command );
 					return sprintf(
-						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.",
+						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.%s",
 						$child,
 						$parent_name,
-						$parent_name
+						$parent_name,
+						! empty( $suggestion ) ? "\n\nDid you mean '{$suggestion}'?" : ''
```

Comment:
I think we should use `PHP_EOL` over `\n` for best cross-platform compatibility.

Do we have other examples of using two line breaks, or do most errors have only one?